build/localnet/README.md

```diff
@@ -206,6 +206,40 @@ The k8s manifests that `tilt` submits to the cluster can be found in [this direc
 
 Tilt continuously monitors files on local filesystem in [specific directories](Tiltfile#L27), and it rebuilds the binary and distributes it to the pods on every code change. This allows developers to iterate on the code and see the changes immediately (i.e. hot-reloading).
 
+## Debug with dlv
```

Comment:
Make sure the TOC is updated at the top.

See https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one to make your life easier